curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67333082ab35e414eb4ea70d.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 67333082ab35e414eb4ea70d
+title: Step 14
+challengeType: 0
+dashedName: step-14
+---
+
+# --description--
+
+When the `a` elements are hovered, the color of the background makes a transition to a different color. You can regulate how that transition happen with the `transition` property:
+
+```css
+a {
+  transition: color 1s linear;
+}
+```
+
+The values that the `transition` property accepts are, in order, the property that the transition is applied to, the duration of the transition, and then the timing.
+
+If there are multiple properties that have a transition, you can write the values for each separated by a comma:
+
+```css
+p {
+  transition: property1 0.1s, property2 0.6s linear;
+}
+```
+
+If a value is omitted, like the timing for the first property, the defaul value is applied.
+
+Add to the `.card` selector `transition: transform 0.3s, background-color 0.3s ease`.
+
+Try it out, the hover transition is complete.
+
+# --hints--
+
+The `.card` selector should have a property `transition` set to `transform 0.3s, background-color 0.3s ease`.
+
+```js
+assert.strictEqual(
+  new __helpers.CSSHelp(document).getStyle(".card")?.getPropertyValue("transition"),
+  "transform 0.3s ease 0s, background-color 0.3s ease 0s" // this is what comes out of `getPropertyValue`
```

Comment:
Not able to pass this test. Here's the value I got: `transform 0.3s, background-color 0.3s`